server/src/main/java/org/apache/cassandra/sidecar/modules/CassandraOperationsModule.java

```diff
@@ -46,6 +49,13 @@
  */
 public class CassandraOperationsModule extends AbstractModule
 {
+    @ProvidesIntoMap
+    @KeyClassMapKey(TableSchemaMapKeys.SystemViewsClientsSchemaKey.class)
+    TableSchema systemViewsClientsSchema(ConnectedClientsSchema connectedClientsSchema)
+    {
+        return connectedClientsSchema;
+    }
```

Comment:
now including `ConnectedClientsSchema` as part of the `SidecarSchemaInitializer`